### PR TITLE
Improve Slack notifications & respect provided timeout value

### PR DESCRIPTION
**Pull request by indrekj**
INF-2932

### CODE REVIEW

File: src/com/salemove/deploy/Notify.groovy [hunk 1]
```diff
@@ -159,4 +174,11 @@ class Notify implements Serializable {
     "&version=${rollbackVersion}" +
     "&environment=${env.kubeEnvName}"
   }
+
+  private def helmRollbackURL(env, rollbackVersion) {
+    'https://jenkins.salemove.com/job/deploy/job/rollback-helm-deploy/parambuild/' +
```
Comment (take-five) [inline src/com/salemove/deploy/Notify.groovy:179]: jenkins.glia.com

File: src/com/salemove/deploy/GoDuration.groovy [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package com.salemove.deploy
+
+import com.cloudbees.groovy.cps.NonCPS
+
+class GoDuration {
```
Comment (take-five) [inline src/com/salemove/deploy/GoDuration.groovy:5]: Why is called GoDuration?

**Review by take-five — APPROVED** [review]

File: src/com/salemove/deploy/Notify.groovy (hunk 1 above)
Comment (indrekj) [reply src/com/salemove/deploy/Notify.groovy]: fbfc928

File: src/com/salemove/deploy/GoDuration.groovy (hunk 2 above)
Comment (indrekj) [reply src/com/salemove/deploy/GoDuration.groovy:5]: https://helm.sh/docs/intro/using_helm/

> --timeout: A [Go duration](https://golang.org/pkg/time/#ParseDuration) value to wait for Kubernetes commands to complete. This defaults to 5m0s.

**Comment by sm-deployer** [comment]
@indrekj, the changes were validated in acceptance. Please click **Proceed** [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-144/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-144/4//console)) to continue the deployment.

**Comment by sm-deployer** [comment]
@indrekj, the changes have been deployed to beta. Please continue [here](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-144/4/display/redirect) (or [in the old UI](https://ci.at.samo.io/job/salemove/job/pipeline-lib/job/PR-144/4//console)).

**Pull request merged**